crates/node-e2e-tests/tests/it/eth.rs

```diff
@@ -116,6 +116,97 @@ async fn can_run_eth_node() -> eyre::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn can_run_eth_node_with_auth_engina_api_over_ipc() -> eyre::Result<()> {
+    let tasks = TaskManager::current();
+    let test_suite = TestSuite::new();
+
+    // Node setup
+    let node_config = NodeConfig::test()
+        .with_chain(test_suite.chain_spec.clone())
+        .with_rpc(RpcServerArgs::default().with_http());
+
+    let NodeHandle { mut node, node_exit_future: _ } = NodeBuilder::new(node_config)
+        .testing_node(tasks.executor())
+        .node(EthereumNode::default())
+        .launch()
+        .await?;
+
+    // setup engine api events and payload service events
+    let mut notifications = node.provider.canonical_state_stream();
+    let payload_events = node.payload_builder.subscribe().await?;
+
+    // push tx into pool via RPC server
+    let eth_api = node.rpc_registry.eth_api();
+    let transfer_tx = test_suite.transfer_tx();
+    eth_api.send_raw_transaction(transfer_tx.envelope_encoded()).await?;
+
+    // trigger new payload building draining the pool
+    let eth_attr = eth_payload_attributes();
+    let payload_id = node.payload_builder.new_payload(eth_attr.clone()).await?;
+
+    // resolve best payload via engine api
+    let client = node.engine_ipc_client();
+
+    // ensure we can get the payload over the engine api
+    let _payload = client.get_payload_v3(payload_id).await?;
+
+    let mut payload_event_stream = payload_events.into_stream();
+
+    // first event is the payload attributes
+    let first_event = payload_event_stream.next().await.unwrap()?;
+    if let reth::payload::Events::Attributes(attr) = first_event {
+        assert_eq!(eth_attr.timestamp, attr.timestamp);
+    } else {
+        panic!("Expect first event as payload attributes.")
+    }
```

Comment:
we can end the test here, no need to duplicate the other code.
we checked that we can send FCU over ipc which is sufficient.